CHANGELOG.md

```diff
@@ -10,3 +10,14 @@ The following sections list the changes for 1.1.0
 - Moved files for upload from web acceptance directory [#71](https://github.com/owncloud/owncloud-test-middleware/pull/71)
 - Add functions related to locking file [#82](https://github.com/owncloud/owncloud-test-middleware/pull/82)
 - Improve starlark: [#83](https://github.com/owncloud/owncloud-test-middleware/pull/83)
+
+## Changelog for [1.2.0] (2022-01-18)
```

Comment:
The text looks good. Should we put the latest changelog at the top, like other changelogs:
https://github.com/owncloud/ocis/blob/master/CHANGELOG.md